helm/values.yaml

```diff
@@ -0,0 +1,140 @@
+worker:
+  replicaCount: 1
+  name: worker
+  image:
+    repository: docker.pkg.github.com/pinterest/querybook/querybook
```

Comment:
added official repo here https://hub.docker.com/r/querybook/querybook, can you please update

Reply:
Yay, finally! Thank you so much :)